src/clr-angular/popover/dropdown/_menu-mixins.clarity.scss

```diff
@@ -78,12 +79,9 @@
 }
 
 @mixin generate-dropdown-item-height() {
-
-    height: $clr-dropdown-item-height;
-    line-height: $clr-dropdown-item-height;
+    padding: 0.125rem 1rem;
 
     @media screen and (max-width: map-get($grid-breakpoints, sm)) {
-        height: $clr-dropdown-item-height-sm;
-        line-height: $clr-dropdown-item-height-sm;
+        padding: 0.25rem 1rem;
```

Comment:
Shouldn't these be variables?

And I can't really check, but if the `$clr-dropdown-item-height` variable isn't used anywhere anymore, you should delete it.

Reply:
The dropdowns CSS doesn't have any variables. We need to fix that but I thought I should do that in a separate PR and not pollute this one.